client/src/pages/spools/create.tsx

```diff
@@ -27,7 +27,7 @@ export const SpoolCreate: React.FC<IResourceComponentsProps & CreateOrCloneProps
     formProps.initialValues.filament_id = formProps.initialValues.filament.id;
   }
 
-  const { queryResult } = useSelect<IFilament>({
+  const {queryResult} = useSelect<IFilament>({
```

Comment:
Your prettier is making changes like this all over the place. Is it configured correctly?

Reply:
ok, i cant work out how to configure it properly. there is a `.prettierrc` file but it dosnt have anything about spaces inside braces. could you check that the `.prettierrc` file in the repo is up to date with the config your using?